amr-wind/wind_energy/ABLStats.H

```diff
@@ -95,6 +98,8 @@ private:
     SecondMomentAveraging m_pa_tu;
     SecondMomentAveraging m_pa_uu;
     ThirdMomentAveraging m_pa_uuu;
+    ScratchFieldPlaneAveraging* m_pa_sfs;
+    ScratchFieldPlaneAveraging* m_pa_tsfs;
```

Comment:
```suggestion
    std::unique_ptr<ScratchFieldPlaneAveraging> m_pa_sfs;
    std::unique_ptr<ScratchFieldPlaneAveraging> m_pa_tsfs;
```

Although I don't think these need to be instance variables.

Reply:
Tried addressing in https://github.com/Exawind/amr-wind/pull/290/commits/22cbdfe875f7c910a3fd1369c871a361c06ac0dd